lang/chicken-scheme/Makefile

```diff
@@ -8,13 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=chicken-scheme
-PKG_VERSION:=5.2.0
-PKG_RELEASE:=2
```

Comment:
This was removed why?

Reply:
because the new version is 5.4.0...

Reply:
I am asking about PKG_RELEASE, why it was removed. The reply is imho incorrect. You might want to look at README in this repository.

Reply:
Ok, fixed `PKG_RELEASE`, it's 1 now.